src/apps/lwaftr/binding_table.lua

```diff
@@ -430,7 +430,6 @@ local function parse_br_addresses(parser)
       parser:skip_whitespace()
       if parser:check(',') then parser:skip_whitespace() end
    end
-   if #addresses == 0 then parser:error('no lwaftr addresses specified') end
```

Comment:
Does it work to do ffi.new('int[?]', 0) ?  What does it return?  If it does not throw an exception, lgtm.